robotiq_c_model_control/launch/single_modbus_gripper.launch

```diff
@@ -0,0 +1,4 @@
+<launch>
+  <arg name="log_output" default="log" />
+  <node pkg="robotiq_c_model_control" type="autodetect_gripper_node.py" name="auto_rtu_node" output="$(arg log_output)" />
```

Comment:
When I first read `autodetect` I thought it was between 'single' and 'dual'. Since both appear to do type detection would suggest migrating to either
*  `autodetect_gripper_node.py` -> `single_gripper_node.py`
or
*    `autodetect_gripper_node.py` -> `detectsingle_gripper_node.py`
*    `dual_gripper_node.py` -> `detectdual_gripper_node.py`
with `name`s to follow a similar pattern

Reply:
Seems reasonable. I tweak it a bit and go for `detect_single_gripper_node.py` and `detect_dual_gripper_node.py`

Reply:
I'm going to leave this for develop. https://readyrobotics.atlassian.net/browse/RPI-32